jpegenc/subblocks/quantizer/quantiser.py

```diff
@@ -0,0 +1,154 @@
+"""The above module is the hardware implementation
+    of quantizer top module"""
+
+from myhdl import always_seq, block, always_comb
+from myhdl import Signal, concat, modbv, intbv
+
+from .ramz import ramz
+from .quantiser_core import (quantizer, QuantOutputStream,
+                             QuantInputStream, QuantConfig)
+
+
+class QuantIDataStream(object):
+    """
+    Input datastream into the Quantizer top module
+
+    data_in : send input data into the module
+    read_addr : read the data from the input buffer
+
+    """
+    def __init__(self, width, addr):
```

Comment:
`addr` doesn't make any sense as a constructer argument.  I can look at the code and see it is the width of the read_addr attribute .